cloud/security/strict-ssl.md

```diff
@@ -0,0 +1,78 @@
+---
+title: Connect with a stricter SSL mode
+excerpt: Connect to Timescale Cloud with a stricter SSL mode
+product: cloud
+keywords: [security]
+tags: [ssl]
+---
+
+# Connect with a stricter SSL mode
+
+The default connection string for Timescale Cloud uses the SSL mode `require`.
+If you want more security, you can connect with a SSL mode of `verify-ca` or
+`verify-full`. To do so, you need to store a copy of the certificate chain where
+your connection tool can find it.
```

Comment:
I wonder if we should be linking to the list of `sslmode` options here https://www.postgresql.org/docs/current/libpq-ssl.html#LIBPQ-SSL-PROTECTION